integration-tests/src/test_loop/tests/state_sync.rs

```diff
@@ -95,7 +96,22 @@ fn setup_initial_blockchain(
             }
         })
         .collect::<Vec<_>>();
-    let clients = validators.iter().map(|v| v.account_id.clone()).collect::<Vec<_>>();
+    let mut clients = validators.iter().map(|v| v.account_id.clone()).collect::<Vec<_>>();
+
+    if let Some(sched) = extra_node_shard_schedule.as_ref() {
```

Comment:
nit: schedule 

Because it's more readable but also the spell check will be unhappy ;)

Reply:
done